test/integration/aggregate.test.ts

```diff
@@ -358,8 +358,13 @@ describe(
 );
 
 async function waitForSearchIndexing(): Promise<void> {
-  const { aggs } = await xata.db.teams.aggregate({ total: { count: '*' } });
-  if (aggs.total !== teams.length) {
+  try {
+    const { aggs } = await xata.db.teams.aggregate({ total: { count: '*' } });
+    if (aggs.total !== teams.length) {
```

Comment:
If we negate the condition, we can move the setTimeout and the recursive call outside the try/catch and de-duplicate the retry code (the catch can be left empty with a //noop comment so that the linter doesn't complain).

```suggestion
    if (aggs.total === teams.length) {
      return;
```